hack/update-bazel.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+# Copyright 2018 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -euo pipefail
+
+bazel run //:gazelle
```

Comment:
We may need to install bazel or make sure bazel is already installed?

Reply:
installing bazel is not a good idea, as it will do some modification to user's development environment. In CI environment the container image does contain the bazel binary. 
If bazel is not installed, the script is simple enough and just throw error like `bazel not found` and exit. which should be easy for users to find what's wrong.